pkg/api/update.go

```diff
@@ -185,16 +185,23 @@ func (s *server) update(w http.ResponseWriter, r *http.Request) {
 			s.writeResponse(w, http.StatusConflict, generateEmptyResponse())
 			return
 		}
-	} else if instance.Status != service.InstanceStateProvisioned {
-		log.WithFields(logFields).Debug(
-			"bad updating request: the instance to update to is not in a " +
-				"provisioned state",
-		)
-		// The instance to update is not in a provisioned state
-		// krancour: Choosing to interpret this scenario as unprocessable
-		// TODO: Write a more detailed response
-		s.writeResponse(w, http.StatusUnprocessableEntity, generateEmptyResponse())
-		return
+	} else {
```

Comment:
i rebased off my other open PR and this came with it, so review this in the other one.